frontend/Auth/Login.tsx

```diff
@@ -1,9 +1,167 @@
+          Sign in
+        </h1>
+        <br />
+        <Form onSubmit={unary(submitMutation.mutate)}>
+          <Form.Field>
+            <label
+              htmlFor="username"
+              style={{
+                fontFamily: "Source Sans Pro",
+                color: "#403F3D",
+                fontWeight: 600,
+              }}
+            >
+              Email
+            </label>
+            <input
+              id="username"
+              type="email"
+              className="ui input"
+              value={loginFormData.username}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  username: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <Form.Field>
+            {/* // @ts-ignore  */}
+            <label
+              htmlFor="password"
+              style={{
+                fontFamily: "Source Sans Pro",
+                color: "#403F3D",
+                fontWeight: 600,
+              }}
+            >
+              Password
+            </label>
+            <input
+              id="password"
+              type="password"
+              className="ui input"
+              value={loginFormData.password}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  password: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <div className="py-8"></div>
+
+          <div className="flex grid grid-cols-2 gap-4 space-x-4">
+            <Button
+              type="submit"
+              style={{
+                background: "#2a467b",
+                color: "white",
+              }}
+              className="h-15 rounded font-medium flex-shrink"
+            >
+              Sign in
+            </Button>
+            <Link
+              to="/reset-password"
+              className="h-15 rounded self-center text-center font-medium text-blue-900 bg-white flex-shrink"
+            >
+              Reset Password
+            </Link>
+          </div>
+
+          <div className="pt-6 pb-2 flex flex-row">
+            <hr className="w-full mt-4 mb-8 border-gray-400"></hr>
+            <span className="bg-white py-2 px-2 text-gray-400 text-sm">or</span>
+            <hr className="w-full mt-4 mb-8 border-gray-400"></hr>
+          </div>
+
+          <Button className="ui grey basic button w-full" type="button">
+            Insert Google Button here
+          </Button>
+
+          <p className="text-center text-xs text-gray-600 py-8">
+            New to Flax?{" "}
+            <Link href="/create-account" className="underline text-blue-900">
+              Create an account
+            </Link>
+          </p>
+        </Form>
+      </div>
+
+      <div className="flex items-center"></div>
```

Comment:
What is this container for?

Reply:
I believe Tailwind requires a parent div set to 3 columns and 3 child divs to create the style equivalent of a Semantic UI card.   I added these divs as part of the conversion to Tailwind.  Let me know if there is a better way to do this.  Thank you.